os-server/src/main/java/app/openschool/course/api/dto/CourseInfoDto.java

```diff
@@ -0,0 +1,91 @@
+package app.openschool.course.api.dto;
+
+import java.util.Set;
+
+public class CourseInfoDto {
```

Comment:
let's have the following approach when the field is mandatory then it is initialized by the constructor, if it is not with the setter method. Now we have both